README.md

```diff
@@ -1,86 +1,99 @@
-## FusionAuth Load Tests ![semver 2.0.0 compliant](http://img.shields.io/badge/semver-2.0.0-brightgreen.svg?style=flat-square)
+# FusionAuth Load Tests ![semver 2.0.0 compliant](http://img.shields.io/badge/semver-2.0.0-brightgreen.svg?style=flat-square)
 
-## Examples Usages:
+This repo contains load tests you can run against any FusionAuth instance.
 
-### Build
 
-See the Savant setup below if you do not yet have Savant configured. 
+## Setup
 
-```
-sb int
-```
+### Start FusionAuth
 
-### Setup FusionAuth
+You will need a working FusionAuth instance. If you want to run load tests, you have probably already figured this part out.
 
-If you want to run load tests, you have probably already figured this part out.
+### Configure FusionAuth
 
-### Configure FusionAuth for load tests
- 
-All of this is optional and will depend upon what you want your tests to do. However, to use the provided examples here, start with these setup steps. 
+#### Create an API Key.
 
-1. Setup an API key. 
-   - If you're a FusionAuth developer, run `sb config` which will insert an API key into the database. 
-   - If you're just an everyday average FusionAuth enthusiast, add an API key however you know how. The examples will expect an API key of `bf69486b-4733-4470-a592-f1bfce7af580` but this can be modified to your liking. 
```

Comment:
I think you are losing some important context here. This tool is really only for us despite being open source, and that uuid (bf69*) is prepopulated into our local dev environments with our kickstart file. 

In reality most of us do not actually want to create a new api-key for this tool, because the defaults are set to what we want

Reply:
That's fine, I can set that as the default like I did with tenant ID

Reply:
>This tool is really only for us despite being open source,

We do document this publicly: https://fusionauth.io/docs/operate/secure-and-monitor/monitor#load-testing